open-machine-learning-jupyter-book/assignments/ml-fundamentals/classification_introduction.ipynb

```diff
@@ -0,0 +1,28 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": 3
+  },
+  "orig_nbformat": 2
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# Delicious Asian and Indian Cuisines "
```

Comment:
Only first letter supposes to be in uppercase. And let's make the file name to be consistent with the content title `delicious-asian-and-indian-cuisines`